hr_employee_calendar_planning/models/hr_employee.py

```diff
@@ -24,23 +59,47 @@ def _regenerate_calendar(self):
                         "name": _("Auto generated calendar for employee")
                         + " %s" % self.name,
                         "attendance_ids": [],
+                        "two_weeks_calendar": two_weeks,
                     }
                 )
                 .id
             )
         else:
             self.resource_calendar_id.attendance_ids.unlink()
-        vals_list = []
+            self.resource_calendar_id.two_weeks_calendar = two_weeks
+        seq = 0
         for line in self.calendar_ids:
             for attendance_line in line.calendar_id.attendance_ids:
+                if attendance_line.display_type == "line_section":
+                    continue
+                duplicate = two_weeks and not line.calendar_id.two_weeks_calendar
+                week_odd = attendance_line.week_type == "1"
                 data = attendance_line.copy_data(
                     {
                         "calendar_id": self.resource_calendar_id.id,
                         "date_from": line.date_start,
                         "date_to": line.date_end,
+                        "week_type": "0" if duplicate else attendance_line.week_type,
+                        "sequence": seq + 26 if week_odd else 24 - seq,
                     }
                 )[0]
+                seq += 1
                 vals_list.append((0, 0, data))
+                if duplicate:
+                    data = attendance_line.copy_data(
+                        {
+                            "calendar_id": self.resource_calendar_id.id,
+                            "date_from": line.date_start,
+                            "date_to": line.date_end,
+                            "week_type": "1",
+                            "sequence": seq + 26,
```

Comment:
Why 26? Use an even bigger hoop (100 or 1000) for being sure (and for having better numbers: 1-101, 2-102...).

Reply:
Odoo sets the sequence of the Odd Week label line to 25 so I did the same and then forced odd week lines to have a sequence greater than 25. but I can change it to 100 if you prefer.

https://github.com/odoo/odoo/blob/13.0/addons/resource/models/resource.py#L266

Reply:
OK, you can put this as comment for later references and let it this way.